### PR TITLE
Move to pdfkit from react-pdf/renderer: BerH Checklist

### PR DESCRIPTION
**Pull request by rbrtrfl**
- refactor BerH Checklist / Merkblatt (former Handout) to be created with `pdfkit` by using `pdfFromUserData`
- unify text of "Druckvermerk" and "Stamp"
- adapt `createPageNumber` from ZOV and adapt corresponding test

- adapt `beratungshilfePdfFromUserdata` tests

### CODE REVIEW

File: app/domains/fluggastrechte/services/pdf/sections/createPageNumber.ts [hunk 1]
```diff
@@ -5,25 +5,35 @@ import {
...
 
+const FONT_SIZE = 7;
+const FONT_HEIGHT_TO_WIDTH_RATIO = 0.6;
+
 export const createPageNumber = (
   doc: typeof PDFDocument,
   footerSect: PDFKit.PDFStructureElement,
   currentPage: number,
   totalPage: number,
+  pageNumberPrefix?: string,
```
Comment (aaschlote) [inline app/domains/fluggastrechte/services/pdf/sections/createPageNumber.ts:16]: let's write an unit test for this

File: app/domains/shared/pdf/createFooter.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import type PDFDocument from "pdfkit";
+import { createPageNumber } from "~/domains/fluggastrechte/services/pdf/sections/createPageNumber";
```
Comment (aaschlote) [inline app/domains/shared/pdf/createFooter.ts:2]: move to shared folder

File: app/domains/shared/pdf/createFooter.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { createStamp } from "~/domains/fluggastrechte/services/pdf/sections/createStamp";
+
+export const createFooter = (
```
Comment (aaschlote) [inline app/domains/shared/pdf/createFooter.ts:5]: make an unit test for this

**Review by aaschlote — APPROVED** [review]
LGTM

File: app/domains/beratungshilfe/services/pdf/checklist/createChecklistDocuments.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  hasGrundeigentum: "Kopie des Grundbuchauszugs von Ihrem Grundeigentum",
+  hasSchwangerschaft: "Angabe des voraussichtlichen Entbindungstermins",
+  hasSchwerbehinderung:
+    "Kopie des Schwerbehindertensuasweis, oder Nachweis über die Behinderung",
+  hasMedicalReasons:
+    "Bescheinigung der medizinischen Notwendigkeit der kostenaufwändigen Ernährung",
+  hasWeitereAusgaben:
+    "Unterlagen, die Ihre Ausgaben belegen, wenn diese nicht auf den Kontoauszügen zu sehen sind",
+} as const satisfies Partial<Record<ReplacementKey, string>>;
+
```
Comment (ekl176) [inline app/domains/beratungshilfe/services/pdf/checklist/createChecklistDocuments.ts:46]: is this mapping identical to the one in app/domains/beratungshilfe/services/pdf/Handout.tsx?

**Pull request merged**